addons/website_blog/data/website_blog_demo.xml

```diff
@@ -77,6 +79,7 @@
             <field name="visits">246</field>
             <field name="cover_properties">{"background-image": "url('/website_blog/static/src/img/cover_2.jpg')", "resize_class": "o_record_has_cover o_half_screen_height", "opacity": "0.4"}</field>
             <field name="content"><![CDATA[
+<section class="s_text_block pt16 pb16" data-snippet="s_text_block"><div class="s_allow_columns o_container_small">
```

Comment:
Something wrong here, you wrap a section in a section (it's valid but seems wrong here)